Packs/CaseManagement-Generic/Scripts/CaseMgmtAnalystTools/CaseMgmtAnalystTools.py

```diff
@@ -0,0 +1,30 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+# requires an XSOAR list that contains a markdown table with links to important Analyst Tools (wikis, google, etc)
```

Comment:
According to my understanding, it is not requires because the default list is returned

Reply:
@israelpoli those are basically the instructions to the user if they want to use a custom list.